src/alto-ui/List/List.scss

```
@@ -70,6 +70,7 @@
 
   &--primary {
     flex: 1;
+    flex-basis: auto;
```

```suggestion
    flex-basis: auto;
```
`Two-value syntax: the first value must be a <number> and it is interpreted as <flex-grow>. The second value must be one of:
a <number>: then it is interpreted as <flex-shrink>.
a valid value for width: then it is interpreted as <flex-basis>.`

---

Changed